src/sdk/model/company/company.ts

```diff
@@ -350,4 +352,16 @@ export class Company extends Entity {
     });
   }
 
+  /**
+   * Get regarding options.
+   * @returns {Array<SupportRegardingOption>}
+   */
+  /* istanbul ignore next: autogenerated */
+  async getRegardingOptions(): Promise<Array<SupportRegardingOption>> {
```

Comment:
lets call it getSupportTicketRegardingOptions and update the TS doc description too